lib/src/code_generator/type.dart

```diff
@@ -39,10 +39,15 @@ abstract class Type {
   /// passed to native code.
   String getCType(Writer w) => throw 'No mapping for type: $this';
 
-  /// Returns the Dart type of the Type. This is the user visible type that is
-  /// passed to Dart code.
+  /// Returns the Dart type of the Type. This is the type that is passed from
+  /// FFI to Dart code.
   String getDartType(Writer w) => getCType(w);
```

Comment:
> * `getCType`
> * `getDartType`
> * `getUserType`

I think I'd prefer the naming as:

* `getCType`
* `getFfiDartType` / `getTrampolineDartType` / `getDartUnwrappedType` / `getInternalDartType`
* `getDartType`

Keeping `getDartType` to be the user visible type.

Reply:
Done